src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -95,9 +109,9 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
 
   const handleBackToEdit = () => {
     const option =
-      selectedAddressKey === "suggestedAddress"
+      selectedAddressKey && selectedAddressKey.includes("suggestedAddress")
```

Comment:
🤔  Was this a bug?

Reply:
Looks like it, but would only affect tracking and still looks like it could be improved. i figured that could come after tests are in place.